client/index.js

```diff
@@ -22,6 +22,10 @@ function getCurrentScriptSource() {
 }
 
 let urlParts;
+let hotReloading = true;
+if (typeof window !== 'undefined') {
+  hotReloading = window.location.search.indexOf('disableAutoReload') === -1;
```

Comment:
not a fan of negative-options, meaning a positive value will result in the negative reassignment of another variable. let's also keep the variable names the same as the querystring parameter. so;

```js
let hotReload = true;
...
const qs = window.location.search.toLowerCase();
hotReload = !qs.indexOf('hotreload=false');
```

the code was trimmed for brevity, but those are the changes I'd like to see

Reply:
Fixed :-)